### ISSUE TITLE
Query Plan Accessibility

### ISSUE THREAD

**Issue by anthonydresser** [Microsoft/sqlopsstudio#1185]
Currently the Query plan library doesn't have accessibility. It shouldn't be to hard to implement it, since it is div based, but it will be a bit of work since there is nothing there currently.

**Comment by kburtram** [comment]
We're going to move the current query plan control into a community extension.
